go/tasks/plugins/array/k8s/config.go

```diff
@@ -32,13 +32,19 @@ var (
 	configSection = config.MustRegisterSection(configSectionKey, defaultConfig)
 )
 
+type TokenConfig struct {
+	primaryLabel string
```

Comment:
These will need to be exported e.g.:
```suggestion
	PrimaryLabel string `json:"primaryLabel" pflag:",What is this?"`
```